mos_tests/murano/test_cli/murano_test.py

```diff
@@ -0,0 +1,163 @@
+#    Copyright 2016 Mirantis, Inc.
+#
+#    Licensed under the Apache License, Version 2.0 (the "License"); you may
+#    not use this file except in compliance with the License. You may obtain
+#    a copy of the License at
+#
+#         http://www.apache.org/licenses/LICENSE-2.0
+#
+#    Unless required by applicable law or agreed to in writing, software
+#    distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+#    WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+#    License for the specific language governing permissions and limitations
+#    under the License.
+
+import json
+from random import randint
+
+import pytest
+
+from tempest_lib import exceptions
+
+pytestmark = pytest.mark.undestructive
+
+
+def rand_num(a=1, b=1000):
+    return randint(a, b)
+
+
+def user_env(user, password, project):
+    """Add this env before you command if you need to perform actions
+    under another user
+    """
+    env = ('env OS_TENANT_NAME="{2}" OS_PROJECT_NAME="{2}" '
+           'OS_USERNAME="{0}" OS_PASSWORD="{1}" ').format(
+            user, password, project)
+    return env
+
+
+def execute_on_all_controllers(env, cmd, fail_ok=False):
+    controllers = env.get_nodes_by_role('controller')
+    all_output = []
+    for controller in controllers:
+        with controller.ssh() as remote:
+            result = remote.execute(cmd)
+            if not fail_ok and not result.is_ok:
+                raise exceptions.CommandFailed(result['exit_code'],
+                                               cmd,
+                                               result.stdout_string,
+                                               result.stderr_string)
+            all_output.append(result)
+    return all_output
+
+
+def wa_for_1543135(env):
+    # WA for "Wrong OS_AUTH_URL makes keystone operations fail"
+    # https://bugs.launchpad.net/mos/+bug/1543135
+    # Changes OS_AUTH_URL from ":5000/" --> ":5000/v2.0" in /root/openrc
+    cmd = unicode(r"sed -i 's#:5000/\x27#:5000/v2.0\x27#g' /root/openrc")
+    execute_on_all_controllers(env, cmd)
+
+
+@pytest.mark.testrail_id('543127')
+@pytest.mark.testrail_id('543125')
```

Comment:
Yes, it covers 2 cases:
https://mirantis.testrail.com/index.php?/cases/view/543127
https://mirantis.testrail.com/index.php?/cases/view/543125